docs/source/pythonapi/base.rst

```diff
@@ -178,6 +178,24 @@ Post-processing
    openmc.StatePoint
    openmc.Summary
 
+Polynomial
+---------------
```

Comment:
I would just remove this header (since it already makes sense under "Post-processing") and replace it with a sentence shortly describing the purpose of the following class/function.